repository/d.json

```diff
@@ -346,6 +346,20 @@
 				}
 			]
 		},
+		{
+			"name": "Debian",
+			"description": "Debian packaging file syntax highlighting for Sublime Text.",
+			"author": "barnumbirr",
+			"details": "https://github.com/barnumbirr/sublime-debian",
+			"issues": "https://github.com/barnumbirr/sublime-debian/issues",
```

Comment:
Since you provide a "details" key that points to a GitHub repo, you don't need to specify "issues", "author" or "description": this information is just taken from the repo, which makes it much easier to update later etc. So, please remove those keys.

Reply:
I didn't know that was pulled automatically, thanks. Done.